### DISCUSSION TITLE
I was wondering if ACE completely replaces GATO as something better?

### DISCUSSION THREAD

**Discussion by ScottzCodez** [daveshap/ACE_Framework#50] [General]
I was wondering if ACE completely replaces GATO as something better?
Are there any aspects of GATO that are worthy to be implemented into ACE?
It's seems ACE is a complete replacement / upgrade to me.

**Comment by DataBassGit** [comment]
Gato was designed to be an organization. This is an autonomous cognitive architecture.

**Comment by daveshap** [comment]
GATO and ACE are completely unrelated. The GATO framework Layer 2 (autonomous agents) promotes the creation of aligned autonomous agents. I abandoned GATO because it got bogged down in process (death by committee). It got too big too fast.
